docs/sources/mimir/references/architecture/components/querier.md

```diff
@@ -46,8 +46,8 @@ The request to the ingesters fetches samples that have not yet been uploaded to
 
 The configured period for `-querier.query-ingesters-within` should be:
 
-- greater than `-querier.query-store-after` and,
```

Comment:
personally I think we should be keeping the `and,`. Other than that, lgtm.

Reply:
Yea, it's a bit subjective, but usually sets of bullet points, including elsewhere in our docs, imply `and`.

Reply:
Perhaps it'd be clearer if the sentence above the dot points was something like "The configured period for `-querier.query-ingesters-within` should be the greater of both:"?

Reply:
Does `should be greater than both:` work the same for you?

Reply:
Yep, sounds good to me.